### PR TITLE
Adding Scaleable Analytics Adapter

### PR DESCRIPTION
**Pull request by cconnors1515**
## Type of change

- [ ] Bugfix
- [ ] Feature
- [ ] New bidder adapter   
- [ ] Code style update (formatting, local variables)
- [ ] Refactoring (no functional changes, no api changes)
- [ ] Build related changes
- [ ] CI related changes
- [ ] Does this change affect user-facing APIs or examples documented on http://prebid.org?
- [x] Other - New Analytics Adapter

## Description of change
Adding Scaleable Analytics Adapter

## Other
Sibling PR: https://github.com/prebid/prebid.github.io/pull/1322

### CODE REVIEW

**Comment by cconnors1515** [comment]
Current failing tests are unrelated to my code

**Comment by cconnors1515** [comment]
@mike-chowla Just checking in on an update with this as it has been 3 weeks and it is now marked as stale.

**Comment by cconnors1515** [comment]
@jaiminpanchal27 @jsnellbaker Is there anyone else that can help me out with this and get it approved, it's been three weeks and I haven't heard a thing while noticing other adapters have been approved and merged already when the PRs were created after mine.

**Comment by jsnellbaker** [comment]
@cconnors1515 I'll take a look at this PR today and follow-up; apologies for the delay.

File: test/spec/modules/scaleableAnalyticsAdapter_spec.js [hunk 1]
```diff
@@ -0,0 +1,139 @@
...
+      });
+    });
+
+    afterEach(function() {
+      events.getEvents.restore();
+      scaleableAnalytics.disableAnalytics();
+    });
+
+    // after(function() {
+    // });
```
Comment (jsnellbaker) [inline test/spec/modules/scaleableAnalyticsAdapter_spec.js]: Some commented code that could be removed.

**Comment by cconnors1515** [comment]
@jsnellbaker Thank you for taking the time to look at this!

I removed the commented code from the spec file and you can test with site param of : 5c4fab7a829e955d6c265e72

**Comment by jsnellbaker** [comment]
@cconnors1515 Thanks for the update and the test site.  Everything worked fine when I tested again.

**Comment by cconnors1515** [comment]
@jsnellbaker Thanks for all your help! Are you able to merge my sibling PR for the docs as well or does bretg have to do that?

**Pull request merged**